articles/read_a_geotiff_with_gdal_in_python.md

```diff
@@ -0,0 +1,39 @@
+# Reading GeoTiff raster images in python3
+
+| [Gokul Ganesan](https://www.linkedin.com/in/gokul-ganesan/) |&nbsp;<a href="https://twitter.com/Fa7C0n"><img alt="SVG" src="/icons/Twitter_Social_Icon_Circle_Color.svg" width="17px" height="17px"> &nbsp;<a href="https://www.linkedin.com/in/gokul-ganesan/"><img alt="PNG" src="/icons/icons8-linkedin.svg" width="20px" height="20px"> &nbsp;[:octocat: ](https://github.com/Fa7C0n) &nbsp;[:email: ](mailto:gokulvganesan@gmail.com)|2020-01-04|
+
+With the constant increase in the volume of earth observation data from satellites and also from aerial platforms like aircrafts and drones it is inevitable that we need use a programming language to automate some of the repetitive tasks. In the recent years python has become the defacto language for data science and machine learning mainly due to its simple structre and a very rich library collections, so in the following article I tried to explain the first step towards automating a raster processing workflow (i.e) reading a raster data.
+
```

Comment:
Give a very brief explanation what is goal and rasterise. Maybe you can provide links of installing those two packages

Reply:
I have done this ... added links to the GDAL and rasterio documentation page.